composer.json

```diff
@@ -99,8 +99,14 @@
     },
     "autoload": {
         "psr-4": {
-            "yii\\": "framework/",
-            "yii\\cs\\": "cs/src/"
+            "yii\\": "framework/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "yii\\cs\\": "cs/src/",
+            "yii\\build\\": "build/",
+            "yiiunit\\": "tests/"
```

Comment:
Why do we need PSR-4 autoload for these? It worked well before.

Reply:
@samdark why do we need a third-party autoloader if Yii  have own autoloader? :) Mainly to shut up IDE's